packages/Liquid-Tests.package/LQLoadTest.class/instance/testTwoHundredUsers.st

```diff
@@ -15,14 +15,14 @@ testTwoHundredUsers
 	pollId := 'gotta-catch-rayquaza'.
 	poll
 		startWithId: pollId
-		ifTaken: [self error: 'this shouldnt have occured'].
+		ifTaken: [self fail].
 		
 	"participant"
 	target := 200.
 	target timesRepeat: [
 	p := PollRepo default
 				at: pollId
-				ifAbsent: [self error: 'nonono'].
+				ifAbsent: [self fail].
```

Comment:
[sand] Maybe use self fail: aMessage instead of fail to make it easier to find out what's wrong (error message will be displayed in the github CI log)

Reply:
+1 from me too, I will add some messages to all occurrences of fail